api/compose/api.go

```diff
@@ -175,4 +183,14 @@ type Stack struct {
 // LogConsumer is a callback to process log messages from services
 type LogConsumer interface {
 	Log(service, container, message string)
+	Status(service, container, message string)
```

Comment:
Why do we need this callback in addition to the `Listener`, that could include both exit code and some message (or build message client-side, instead of hardcoding this message server-side). 
I just need to understand this a bit more with the various constraints...

Reply:
due to service logs coloring in `logCusumer`

Reply:
yes, makes sense